app/main.py

```diff
@@ -1,7 +1,8 @@
 def copy_file(command: str) -> None:
     command_ls = command.split()
```

Comment:
You can probaly simplify code using assigning multiple variables in one line.

Reply:
You mean `cmd, file_in, file_out = command.split()`?

Reply:
Yah, further we will have easy access to these variables